src/arch/riscv/isa/decoder.isa

```diff
@@ -1660,29 +1660,41 @@ decode QUADRANT default Unknown::unknown() {
                         auto sign = bits(unboxF32(Fs2_bits), 31);
                         Fd_bits = boxF32(insertBits(unboxF32(Fs1_bits), 31,
                                                     sign));
+                        status.fs = 3;
```

Comment:
@tastynoob Is it possible to update status register in templates of floating point instructions instead of modify every instructions?